src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2382,39 +2383,51 @@ def delete_ssl_cert(cmd, resource_group_name, certificate_thumbprint):
     raise CLIError("Certificate for thumbprint '{}' not found".format(certificate_thumbprint))
 
 
-def import_ssl_cert(cmd, resource_group_name, name, key_vault, key_vault_certificate_name):
+def import_ssl_cert(cmd, resource_group_name, name, certificate_name=None, key_vault_certificate_name=None,
```

Comment:
if this is the only parameter we expect for ASC - then this can ASC resource_id as well since ASC can be in a different subscription than the subscription the app is in.

Reply:
okay, makes sense, I'll change the parameter, i will need to test this